libcalico-go/lib/backend/model/keys.go

```diff
@@ -722,3 +730,12 @@ func SerializeValue(d *KVPair) ([]byte, error) {
 	}
 	return json.Marshal(d.Value)
 }
+
+func determinePolicyName(name, tier string) string {
```

Comment:
Should this be part of `parseMetadataAnnotation`? It seems like it must always be called at the same time as that function, so having them separately seems error prone?